### ISSUE TITLE
Is there any version in Tenserflow

### ISSUE THREAD

**Issue by sarraghd** [sshaoshuai/PointRCNN#69]
Hi, 
Thank you for this great job, but i wonder if there is any version with tensorflow?
Thank you

**Comment by sshaoshuai** [comment]
No, this work is implemented with PyTorch and we didn't use TensorFlow in this code except Tensorboard.
